frontend/src/components/organisms/AddingPlayerForm/AddingPlayerForm.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Button, TextField, Typography } from "@material-ui/core";
+import { Centered } from "../../atoms/Shared/Centered";
+import { VerticalSpace } from "../../atoms/Shared/VerticalSpace";
+
+function AddingPlayerForm() {
+  return (
+    <Centered>
+      <VerticalSpace height="1rem" />
+      <Typography component="h6" variant="h6">
+        Nowy zawodnik
+      </Typography>
+      <VerticalSpace height="1rem" />
+
+      <form>
+        <Centered>
+          <TextField
+            key={"name"}
+            id={`outlined-name`}
+            label={"Imię"}
+            variant={`outlined`}
+          />
+          <VerticalSpace height="1rem" />
+
+          <TextField
+            key={"surname"}
+            id={`outlined-surname`}
+            label={"Nazwisko"}
+            variant={`outlined`}
+          />
+          <VerticalSpace height="1rem" />
+
+          <TextField
+            key={"email"}
+            id={`outlined-email`}
+            label={"Adres e-mail"}
+            variant={`outlined`}
+          />
+          <VerticalSpace height="1rem" />
+
+          <TextField
+            key={"phone"}
+            id={`outlined-phone`}
+            label={"Numer telefonu"}
+            variant={`outlined`}
+          />
+          <VerticalSpace height="1rem" />
+        </Centered>
+
+        <Centered>
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={() => {
+              console.log("zapisz zawodnika");
+            }}
+          >
+            Zapisz zawodnika
+          </Button>
+          <VerticalSpace height="1rem" />
+        </Centered>
+      </form>
+    </Centered>
```

Comment:
To co masz jest OK :)
Ale wyrzuciłbym taką część do atomic design:
`
<Centered>
<TextField
            key={"email"}
            id={`outlined-email`}
            label={"Adres e-mail"}
            variant={`outlined`}
          />
          <VerticalSpace height="1rem" />
          </Centered>`

To w katalogu atomic :)

Bo powtarzałoby się 
        `  <VerticalSpace height="1rem" />  ` oraz  `variant={`outlined`}` a w przyszłości może i stylowanie

Reply:
Czy te dane są już gdzieś zapisywane w jakiś state'sach?

Reply:
Zastanawiałem się nad tym Atmociem - dzięki za szybki komentarz.

Na razie jeszcze danych nie przechwytuję - in progress :-)

Reply:
Nie przesadzajmy z tym Atomic :) Jak dla mnie to TextField jest juz atomic sam w sobie.